### DISCUSSION TITLE
Connectivity issue after converting the IoT hub to DigiCert.

### DISCUSSION THREAD

**Discussion by Navyadgk** [Azure/azure-iot-sdk-java#1711] [Q&A]
Hi @timtay-microsoft ,

Recently, we created a new IoT hub and migrated it from Baltimore to DigiCert.

The messages are received from IoT devices to Cloud using Mqtt.

We have followed the procedure for updating certificates in the device too.

The reporting interval is 13min. i.e., the device should report every 13min.

For Ex:

>>For the first D2C messages, the date is successfully published and received after 13min of the device start.

>> The second D2C message is failing to publish from the device side(i.e., checked the logs it is showing mqtt connected but failed to publish the data.

>> This is happening for every alternate reading.

Could you please help me why this is happening? i.e., we are missing a reading every alternate time.

Below is the picture, in which the first column is generated message time from the device and 3rd column is the inserted time into the IOT hub and 4th is the count of messages. As you there is a difference of approx. 30min from one reading to another.

<img width="254" alt="gts" src="https://github.com/Azure/azure-iot-sdk-java/assets/133218325/6c23046f-0bea-4539-a33e-4a991f6389a2">

**Comment by timtay-microsoft** [comment]
Can you provide some SDK-side logs for your device here? And what version of the SDK are you using?

**Reply by timtay-microsoft** [reply]
Also, please note that most older versions of the SDK don't work when the service uses the new Digicert root certificate. You can read more about what you need to do [here](https://github.com/Azure/azure-iot-sdk-java/blob/main/upcoming_certificate_changes_readme.md) if you are using one of these older versions.

**Comment by timtay-microsoft** [comment]
@Navyadgk did you still need help here?

**Reply by Navyadgk** [reply]
Thank you @timtay-microsoft for your reply,  I have mistakenly posted the query in SDK-Java but we are using Azure IoT hub and for connectivity, we are using MQTT protocol.
